sat/cli/jobstat/main.py

```diff
@@ -44,11 +45,16 @@ def wrap_long_row_lines(row):
         None. Modifies row in place.
     """
     if row.get('apid'):
-        row['apid'] = textwrap.fill(text=row['apid'], width=20)
+        row['apid'] = row['apid'][:8]+"..."
     if row.get('jobid'):
         row['jobid'] = textwrap.fill(text=row['jobid'], width=20)
     if row.get('command'):
-        row['command'] = textwrap.fill(text=row['command'], width=40)
+        tempstr = textwrap.fill(text=row['command'], width=40)
+        if re.findall("\A/.*/mpiexec", tempstr):
+            tempstr = re.sub("\A/.*/mpiexec", "mpiexec", tempstr, 1)
```

Comment:
I get `pycodestyle` violations from these regexes:
```
mbp-ekamin:sat ekamin$ pycodestyle --config pycodestyle.conf sat tests
sat/cli/jobstat/main.py:53:24: W605 invalid escape sequence '\A'
sat/cli/jobstat/main.py:54:31: W605 invalid escape sequence '\A'
sat/cli/jobstat/main.py:55:26: W605 invalid escape sequence '\A'
sat/cli/jobstat/main.py:56:31: W605 invalid escape sequence '\A'
```

If you specify as raw strings that will fix the warnings, e.g. using `r'\A/.*/mpiexec...'`

Also, like before, would prefer that we use single quotes to stay consistent.

Getting a little nitpicky but also might suggest that we use `re.match` or `re.search` rather than `re.findall` - `re.findall` returns a list of matches and we don't actually do anything with those, I think what we're trying to do is just see if the start of the string matches the regex so something that returns just a single `match` object might be better.

Note that with `re.match` we don't strictly need the `'\A'` because `re.match` always matches at the start of a string, but if you wanted to use the same regex for both ll 53 and 54 it's probably fine to keep the `\A` character.

Overall I would probably write these lines as:

```suggestion
        mpiexec_command_regex = r'\A/.*/mpiexec'
        if re.match(mpiexec_command_regex, tempstr):
            tempstr = re.sub(mpiexec_command_regex, 'mpiexec', tempstr, 1)
```

Reply:
I am also wondering if we even need to be checking with `re.search`/`re.findall`/`re.match`. Won't `re.sub` just return the original string if the regex is not matched? Why not just this, with no `if` cases?
```
tempstr = re.sub(r'\A/.*/mpiexec', 'mpiexec', tempstr, 1)
tempstr = re.sub(r'\A/.*/aprun', 'aprun', tempstr, 1)
```